src/main/java/org/spongepowered/common/data/manipulator/immutable/ImmutableSpongeMobSpawnerData.java

```diff
@@ -189,39 +188,39 @@ public ImmutableNextEntityToSpawnValue nextEntityToSpawn() {
         return this.toSpawnValue;
     }
 
-    public short getRemaining() {
+    private short getRemaining() {
```

Comment:
Nah, these could also just be removed with lambda notations: `() -> this.remaining`